signalbot/command.py

```diff
@@ -1,10 +1,29 @@
 import functools
+import re
 from abc import ABC, abstractmethod
 
 from .message import Message
 from .context import Context
 
 
+def regex_triggered(*by):
```

Comment:
One more thing, could you add some type hints to this function?

Reply:
I believe I did this right, new to typing in python, if you want to review :pray: